mexico-city/src/main.rs

```diff
@@ -9,7 +9,8 @@
 //! See the `LICENSE.markdown` file in the Veracruz root directory for
 //! information on licensing and copyright.
 
-#![no_main]
+//#![no_main]
```

Comment:
Can this commented out code be deleted?

Reply:
This may be needed for another platform, but I'll need to check (I think TZ need a main.rs but no `fn main` - I need to see why if that's the case, it built and ran successfully on TZ)

Reply:
Could it be a carry over from early quirks of no-std?